pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.pyright]
+include = [
```

Comment:
This requires some discipline to maintain. Do you anticipate our flavor of gradual typing being a series of focused efforts to annotate a given module and then add it here?

Reply:
Yes. That's pretty much what I envision. I use wildcards when I can.

I guess in retrospect maybe I could use opt-out instead of opt-in. I'll probably change it to do that later.